### DISCUSSION TITLE
IRC vs WS

### DISCUSSION THREAD

**Discussion by type2diabetes** [RakambdaOrg/ChannelPointsMiner#234] [General]
Which one _should_ be used if I want to appear in chat for all online streamers and not just the two first in line? Also, does it reconnect to chat if it were to drop?  :)

**Comment by Rakambda** [comment]
From your point of view IRC or WS will do the same, it's just the protocol behind that changes. By default it'll be WS as this is what the twitch website uses.

Both will connect you to chat when a streamer goes online and disconnect when going offline. If connection is dropped, you'll be reconnected. Here's there's no "2" limit, this is just for the channel points mining that it exists.

I don't really know if limitations are different depending on which is used, but I think that in normal use case no limitations will be hit either way.
